modules/gear/loadouts/sf_common.sqf

```diff
@@ -0,0 +1,27 @@
+
+/* Basic Loadout */
+#include "basic.sqf"
+
+/* Weapon & Magazine Slots */
+ADD_WEAPON_ON_BACK("ACE_Coyote_Pack_Black");
+
+/* Optics Slots */
+ADD_WEAPON("NVGoggles");
+ADD_WEAPON("ACE_Rangefinder_OD");
+ADD_WEAPON("ACE_DAGR");
+
+/* Common Grenade Slots */
+ADD_MAGAZINE("ACE_DM51A1", 2);
+ADD_MAGAZINE("SmokeShell", 2);
+ADD_MAGAZINE("ACE_Battery_Rangefinder", 1);
+
+/* Item Slots */
+ADD_WEAPON("ACE_Earplugs");
```

Comment:
earplugs are already given to everyone in a seperate script this will just add duplicate

Reply:
ADD_WEAPON("ACE_Earplugs"); is in each _common.sqf . I didn't put them there. Should they all be removed then?

```suggestion
ADD_WEAPON("ACE_Earplugs");
```
https://github.com/TheCandianVendingMachine/NostalgiaOpsFrameworks/blob/fb0782921bc96a50c23f7cc681f14abc1531082c/modules/gear/loadouts/gue_common.sqf#L12

https://github.com/TheCandianVendingMachine/NostalgiaOpsFrameworks/blob/fb0782921bc96a50c23f7cc681f14abc1531082c/modules/gear/loadouts/ru_common.sqf#L12

https://github.com/TheCandianVendingMachine/NostalgiaOpsFrameworks/blob/fb0782921bc96a50c23f7cc681f14abc1531082c/modules/gear/loadouts/usmc_common.sqf#L12

Reply:
Nope I misunderstood the addearplugs script. They automatically put them in not put them in the inventory. Shouldn't do reviews right when I wake up :P